examples/gnrc_networking/Makefile

```
@@ -36,6 +36,11 @@ USEMODULE += netstats_rpl
 # development process:
 DEVELHELP ?= 1
 
+# set the ZEP port for native (reverse of the default border router configuration)
+ifneq (,$(filter socket_zep,$(USEMODULE)))
+  PORT ?= -z [::1]:17755,[::1]:17754
```

IMHO this shouldn't be `PORT`. It's already pretty murky that the TAP interface with `native` is the `PORT` (but only if there is `netdev_tap`), so adding more blurriness to that variable doesn't make things better (not to talk about the confusing fact that this PORT contains network ports...). Why not just

```suggestion
  TERMFLAGS += -z [::1]:17755,[::1]:17754
```